sormas-e2e-tests/src/test/java/org/sormas/e2etests/steps/web/application/events/EventDirectorySteps.java

```diff
@@ -57,6 +99,81 @@ public EventDirectorySteps(
               dataOperations.getPartialUuidFromAssociatedLink(eventUuid));
         });
 
+    When(
+        "I click checkbox to choose all Event results",
+        () -> {
+          webDriverHelpers.clickOnWebElementBySelector(FIRST_CHECKBOX_EVENT_DIRECTORY);
+          webDriverHelpers.waitForPageLoaded();
+        });
+    And(
+        "I click on Bulk Actions combobox on Event Directory Page",
+        () -> webDriverHelpers.clickOnWebElementBySelector(BULK_ACTIONS_EVENT_DIRECTORY));
+
+    And(
+        "I click on Group Events from Bulk Actions combobox on Event Directory Page",
+        () -> webDriverHelpers.clickOnWebElementBySelector(GROUP_EVENTS_EVENT_DIRECTORY));
+    When(
+        "I navigate to the last created Event page via URL",
+        () -> {
+          String createdEventUUID = CreateNewEventSteps.newEvent.getUuid();
+          String LAST_CREATED_EVENT_PAGE_URL =
+              environmentUrl + "/sormas-webdriver/#!events/data/" + createdEventUUID;
+          webDriverHelpers.accessWebSite(LAST_CREATED_EVENT_PAGE_URL);
+          webDriverHelpers.waitForPageLoaded();
+          webDriverHelpers.waitUntilIdentifiedElementIsVisibleAndClickable(UUID_INPUT, 50);
+          TimeUnit.SECONDS.sleep(5);
+        });
+
+    When(
+        "^I click on ([^\"]*) Radiobutton on Event Directory Page$",
+        (String buttonName) -> {
+          webDriverHelpers.clickWebElementByText(EVENTS_RADIO_BUTTON, buttonName);
+          webDriverHelpers.waitForPageLoaded();
+        });
+    When(
+        "I check that data of linked group is correct",
+        () -> {
+          EventGroup createdGroup = EditEventSteps.groupEvent;
+          EventGroup collectedGroup = collectEventGroup();
+          ComparisonHelper.compareEqualFieldsOfEntities(
+              collectedGroup, createdGroup, List.of("name"));
+          webDriverHelpers.waitForPageLoaded();
```

Comment:
why to we need this if we compare 2 collected entities?

Reply:
removed collectEventGroup() and included getting name of linked group to collectedGroup object. We are comparing only names for this test step.